apps/documentation/src/routes/documentation/index.mdx

```diff
@@ -10,23 +10,23 @@
 
 # Tuono
 
-**Tuono is a fullstack framework for building React applications using Rust as
+**Tuono is a full-stack framework for building React applications using Rust as the
 backend.**
 
 Tuono (Italian word for "thunder", pronounced /2 Oh No/). Why Tuono? Just a badass name.
 
 ## Introduction
 
-**NodeJs/Deno/Bun are the only runtimes that allow a React app to be fullstack right? (no)**
+**NodeJs/Deno/Bun are the only runtimes that allow a React app to be full-stack right? (no)**
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The official spelling of this programming framework is “Node.js”. (NODE_JS[3])
Suggestions: `Node.js`
Rule: https://community.languagetool.org/rule/show/NODE_JS?lang=en-US&subId=3
Category: COMPOUNDING
<!-- __reviewdog__:ChA5MjEzMjQ0M2FhM2FmMjA3EgxMYW5ndWFnZVRvb2w= -->